spec/acceptance/pyvenv_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'spec_helper_acceptance'
+
+describe 'python::pyvenv defined resource' do
+  context 'minimal parameters' do
+    # Using puppet_apply as a helper
+    it 'works with no errors' do
+      pp = <<-EOS
```

Comment:
I think we have a rubocop rule that suggests to use more descriptive name
```suggestion
      pp = <<-PUPPET
```